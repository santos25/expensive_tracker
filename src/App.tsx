import React from 'react';
import { GlobalStyle } from './GlobalStles';
import { Container, Title } from "./AppStyled";
import Main from './components/main/main';
import Total from './components/total/total';

const App: React.FC = (): React.ReactElement => {
  return (
    <Container>
      <GlobalStyle />
      <Title>
        Seguimiento de Gastos
      </Title>
      <Total />
      <Main />
    </Container>
  );
}

export default App;
