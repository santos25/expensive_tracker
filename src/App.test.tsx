import App from './App';
import { shallow } from 'enzyme';

describe('Tests for App.js', () =>{
  it('render app.js', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  })
})