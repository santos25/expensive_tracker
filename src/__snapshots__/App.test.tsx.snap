// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for App.js render app.js 1`] = `
<styled.div>
  <Memo(l) />
  <h1>
    TEsting
  </h1>
</styled.div>
`;
